packages/content-conformance/src/reporters/markdown-reporter.ts

```diff
@@ -43,8 +51,17 @@ export function markdownReporter(files: VFile[]) {
       // TODO: link to descriptive page for the rule
       columns.push(`\`${message.ruleId}\``)
 
-      lines.push(`|${columns.join('|')}|`, '')
+      tableRows.push(`| ${columns.join(' | ')} |`)
     })
+
+    // Global parse errors
+    if (parseErrors.length > 0) {
+      lines.push(...parseErrors, '')
+    }
+
+    // Table
+    lines.push('| Position | Description | Rule |', '|---|---|---|')
```

Comment:
```suggestion
    lines.push('| Position | Description | Rule |', '| --- | --- | --- |')
```

🤔 (needs snapshot update)

Reply:
SIde: Wonder if this GitHub actions `Summary` class is at all useful:
https://github.com/actions/toolkit/blob/main/packages/core/src/summary.ts#L229-L255

Reply:
@thiskevinwang It seems to be working without the spaces here 🤔

And yeah I saw that after I had initially made this! I'm not sure I want to add @actions/core as a dependency here, but I'm tempted.